satellytes/satellytes.com
# feat: replace react-helmet with gatsby head api

This PR replaces the [gatsby-plugin-react-helmet](https://www.gatsbyjs.com/plugins/gatsby-plugin-react-helmet/)(will be deprecated) with the [Gatsby Head API](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-head/)

**Why?**
- gatsby allows build-in `Head` to add element to document head of the pages
- more performant, easier to use than `gatsby-plugin-react-helmet` + `react-helmet`

**How to test:**
- open preview [link](https://deploy-preview-785--satellytes.netlify.app/) and check <head> of the pages
- open preview [link](https://deploy-preview-785--satellytes.netlify.app/) and check SEO data of the pages
